### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by 1ugia**
Updated with:
- team members info
- project info
- function info
- tech info
Added extra:
- project instruction links

Trello Ticket: https://trello.com/c/uJtB4H89/52-chore-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,48 @@
 # AceBook
+## [Makers Academy](http://www.makersacademy.com) - Week 9 Group Project
+Team members:
+* [Merryn Hurley-Rawlins](https://github.com/merrynhr)
+* [Zsuzsanna Ver](https://github.com/MrsVer)
+* [Cynthia Fu](https://github.com/YinnyF)
+* [Fabio Fodrihuez](https://github.com/frodri13)
```
Comment (YinnyF) [inline README.md]: small typo here, should be: Fabio Rodriguez

File: README.md [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+- Sign up
+- Sign in 
+- Sign out
+- Create a post
+- Posts appear with the newest post first
+- View the new post on main home page
+- Like a post
+- Post images
+- CRUD on posts
+- Add comments
```
Comment (YinnyF) [inline README.md:25]: yes yes we do :D

File: README.md [hunk 3]
```diff
@@ -1,15 +1,48 @@
...
+- Nav bar and footer
+- Abouts page
+- Using Facebook Color Palette on CSS: `#3b5998 , #8b9dc3, #dfe3ee, #f7f7f7, 	#ffffff`
 
-1. Fork this repository to `acebook-teamname` and customize
-   the below\*\*
+### Technologies used:
+* Languages - Ruby on Rails, Ruby, HTML, CSS, JavaScript,
```
Comment (YinnyF) [inline README.md:31]: SASS :)

File: README.md [hunk 4]
```diff
@@ -1,15 +1,48 @@
...
-2. The card wall is here:
+## Makers Project Intructions:
 
-[You can find the Trello board here](https://trello.com/b/Sj8Qlv4L/team-soda)
+* [Original Repository at Makers](makersacademy/acebook-rails-template)
+* [Engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
+
+Our card wall is here:
+* [Find our Trello board here](https://trello.com/b/Sj8Qlv4L/team-soda)
+* [Original Trello board from Makers](https://trello.com/b/aN3xtXXl/acebook-rails-template)
```
Comment (YinnyF) [inline README.md]: prob dont include this lol

**Review by YinnyF — CHANGES_REQUESTED** [review]
Thanks for updating the READMe

We also need to mention the following since `bundle exec rake db:migrate` doesn't work anymore:

To migrate the db:
`rails db:migrate`

File: README.md (hunk 1 above)
Comment (frodri13) [reply README.md]: TO BE DONE ASAP!!!

**Review by merrynhr — APPROVED** [review]
Hope thats okay Fabio

**Pull request merged**